src/fusion.cpp

```diff
@@ -108,6 +108,59 @@ miopenStatus_t ConvBiasActivFusion(Handle& handle,
     return miopenStatusSuccess;
 }
 
+static auto AllocateBuffersAndMakeConvBiasActivFusionInvokeParams(
+    const FusionContext& context,
+    const FusionDescription& problem,
+    std::vector<Allocator::ManageDataPtr>& invoke_bufs,
+    miopen::OperatorArgs& params)
```

Comment:
[F.21: To return multiple “out” values, prefer returning a struct or tuple](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f21-to-return-multiple-out-values-prefer-returning-a-struct-or-tuple)

Reply:
tuples must be destroyed along with their inventors.
![stalin_smile_50](https://github.com/ROCmSoftwarePlatform/MIOpen/assets/12697412/2ee0afb3-b6cf-4204-b08f-6f29a9b515a9)